### PR TITLE
Type Store [1.4.2]

### PR DESCRIPTION
**Pull request by Fausto95**
**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-player-store/src/types.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  _id: string,
+  value: string,
+  label: string,
+  items: Array<?string>,
+  media: Media
+|};
+
+type Answers = Array<Array<string>>;
+
+type Content = {|
```
Comment (chrisdugne) [inline packages/@coorpacademy-player-store/src/types.js]: recupere les types de progression-engine et fais les plutot evoluer la bas

File: packages/@coorpacademy-player-store/src/actions/api/analytics.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
+// @flow
```
Comment (chrisdugne) [inline packages/@coorpacademy-player-store/src/actions/api/analytics.js]: // @flow strict

**Comment by chrisdugne** [comment]
[Flow types for player-services](https://trello.com/c/uBiCu3DF/1033-flow-types-for-player-services)

**Comment by chrisdugne** [comment]
[Flow types for player-store](https://trello.com/c/cRTHH0Nk/1035-flow-types-for-player-store)

File: packages/@coorpacademy-player-store/src/actions/ui/progressions.js [hunk 3]
```diff
@@ -77,25 +128,36 @@ export const selectProgression = id => async (dispatch, getState) => {
...
   }
 };
 
-export const OPEN_ASSISTANCE_REQUEST = '@@progression/OPEN_ASSISTANCE_REQUEST';
-export const OPEN_ASSISTANCE_SUCCESS = '@@progression/OPEN_ASSISTANCE_SUCCESS';
-export const OPEN_ASSISTANCE_FAILURE = '@@progression/OPEN_ASSISTANCE_FAILURE';
-
-export const openAssistance = progression => (dispatch, getState, {services}) => {
+export const openAssistance = (progression: Progression) => (
+  dispatch: Function,
```
[the remark this answers is not in the document]
Comment (chrisdugne) [reply packages/@coorpacademy-player-store/src/actions/ui/progressions.js]: agreed, but I had issues with flow using type `Dispatch`, we need to investigate the matter here

File: packages/@coorpacademy-player-store/src/actions/api/analytics.js [hunk 4]
```diff
@@ -1,16 +1,33 @@
...
+  DispatchedAction,
+  Action,
+  GetState,
+  ThunkAction
+} from '../../definitions/redux';
 
-export const MEDIA_VIEWED_ANALYTICS_REQUEST = '@@analytics/MEDIA_VIEWED_REQUEST';
-export const MEDIA_VIEWED_ANALYTICS_SUCCESS = '@@analytics/MEDIA_VIEWED_SUCCESS';
-export const MEDIA_VIEWED_ANALYTICS_FAILURE = '@@analytics/MEDIA_VIEWED_FAILURE';
+export const MEDIA_VIEWED_ANALYTICS_REQUEST: string = '@@analytics/MEDIA_VIEWED_REQUEST';
```
[the remark this answers is not in the document]
Comment (chrisdugne) [reply packages/@coorpacademy-player-store/src/actions/api/analytics.js]: I started to enhance this for once action:
https://github.com/CoorpAcademy/components/pull/1419/files#diff-8e1ddf20e15f755c3469e3935f48c982R43

we need to write this for all of them next

File: packages/@coorpacademy-player-store/package.json [hunk 5]
```diff
@@ -49,6 +49,7 @@
     "rimraf": "^2.6.1"
   },
   "dependencies": {
+    "@coorpacademy/progression-engine": "9.3.0",
```
[the remark this answers is not in the document]
Comment (chrisdugne) [reply packages/@coorpacademy-player-store/package.json]: yes we use types from progression-engine:
https://github.com/CoorpAcademy/components/pull/1419/files#diff-38b9c55ec3280453e823d408424de0d3R4

File: packages/@coorpacademy-player-store/src/actions/api/analytics.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (VincentCATILLON) [reply packages/@coorpacademy-player-store/src/actions/api/analytics.js]: ho ok, not a really nice thing so lol

File: packages/@coorpacademy-player-store/src/actions/ui/progressions.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (VincentCATILLON) [reply packages/@coorpacademy-player-store/src/actions/ui/progressions.js]: ok don't worry it's not a big deal

**Pull request merged**